SPRINT-STRUCTURE.md

```diff
@@ -1,115 +1,74 @@
 # Structure of a Sprint
 
-## Goals
+The purpose of this document is to:
 
-This index structure is intended to accomplish the following goals
+- Organize the content in the playbook for quick reference and discoverability
```

Comment:
nit: remove 'the' 
Organize content in the playbook

Reply:
Fixed